service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateSentinelRunPlaybookAutomationRule.java

```diff
@@ -88,4 +89,9 @@ protected String getResourceType() {
   protected Optional<String> getResourceId(FlightContext context) {
     return Optional.empty();
   }
+
+  @Override
+  public List<ResourceNameRequirements> getResourceNameRequirements() {
+    return List.of();
```

Comment:
Can you refresh my memory, what happens if this automation rule step is replayed and there is already one created?